christfellowshipchurch/web-app-v2
# Cards Width Fix

### About
This PR fixes a bug where the cards on individual season pages for sisterhood podcast were not all the same width. This PR adds a set width to mobile default card. **THIS COULD AFFECT OTHER PAGES - TEST THOROUGHLY**

### Test Instructions
1. Go to the live sisterhood podcast page, scroll down and check any season, we know that seasons 2, 6, and 8 have the issue, and see that on mobile view the cards do not have equal width. [/so-good-sisterhood](https://www.christfellowship.church/so-good-sisterhood)
2. Go to the same page on the testing link [/so-good-sisterhood](https://web-app-v2-git-sgs-card-fix-christ-fellowship-church.vercel.app/?vercelToolbarCode=qJv1VYa81Hfvz22/so-good-sisterhood)

1. Also check clicking see more in the discover page, compare live [/discover](https://www.christfellowship.church/discover) with the [/discover](https://web-app-v2-git-sgs-card-fix-christ-fellowship-church.vercel.app/?vercelToolbarCode=qJv1VYa81Hfvz22/discover) in the testing link.

3. Check that no other page views, that have cards, on mobile view broke. For example `/`, `/so-good-sisterhood`, `/events`, `/get-there-first`, `discover`, and `2023-christ-birthday-offering`.

### Screenshots
|Before|After|
|--|--|
|<img width="1077" alt="Screenshot 2023-09-26 at 3 13 10 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/2c857391-1184-4bfd-9dbe-94dc35e22207">|<img width="1076" alt="Screenshot 2023-09-26 at 3 13 33 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/cb313ccc-4d51-401b-8196-33c07c5d8e0c">|

### Closes Tickets
[CFDP-2912](https://christfellowshipchurch.atlassian.net/browse/CFDP-2912)

[CFDP-2912]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2912?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
